### ISSUE TITLE
Forbid % string interpolation

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#33]
As we have decided to stick to `.format()`, we are banning `f` strings: #32 

But, @vergeev have also suggested to forbid `'string with %s' % some`.
Here's the backstory:
- https://github.com/gforcada/flake8-pep3101/issues/23 has some false positives, that disallow to use regular modulo operator `5 % 2`
- we can only forbid using this operator with string literals, not vars as was suggested in https://github.com/wemake-services/wemake-python-styleguide/issues/32#issuecomment-402086977

So, this would be valid:

```python
some_variable % 'value1'
5 % 2
```

This would be invalid:

```python
'format me %s' % 'value2'
```
